src/PileupTrack.js

```diff
@@ -192,11 +194,13 @@ const PileupTrack = (HGC, ...args) => {
         baseUrl = options.workerScriptLocation;
       }
 
-      const worker = spawn(
-        new Worker('./bam-fetcher-worker.js', {
-          _baseURL: baseUrl,
-        }),
-      );
+      // const worker = spawn(
+      //   new Worker('./bam-fetcher-worker.js', {
+      //     _baseURL: baseUrl,
+      //   }),
+      // );
```

Comment:
Maybe remove? It's in the git history anyway

Reply:
Yup, good point. I can also get rid of the `getThisScriptLocation` function.